docker/package/tezos_voting_wizard.py

```diff
@@ -0,0 +1,559 @@
+            for o in self.options:
+                description = textwrap.indent(
+                    textwrap.fill(self.options[o], 60), " " * 31
+                ).lstrip()
+                if def_i is not None and i == def_i:
+                    print(str_format.format(i, o + " (default)", description))
+                else:
+                    print(str_format.format(i, o, description))
+                i += 1
+
+
+# Steps
+
+network_query = Step(
+    id="network_mode",
+    prompt="Which Tezos network would you like to use?\n",
+    help="The selected network will be used for voting.\n"
+    "Usually you would vote on the mainnet, but you can also select\n"
+    "a custom network (needs to already be set up).",
+    options=networks,
+    validator=Validator(enum_range_validator(networks)),
+)
+
+custom_network_query = Step(
+    id="custom_network_name",
+    prompt="What's the name of the custom network you'd like to use?",
+    help="The selected network will be used to vote.\n"
+    "For example, to use the tezos-baking-custom@voting service, input 'voting'."
+    "You need to already have set up the custom network using systemd services.",
+    # TODO: "\nYou can follow a tutorial here:",
+    validator=Validator(required_field_validator),
+)
+
+new_proposal_query = Step(
+    id="new_proposal_hash",
+    prompt="Provide the hash for your newly submitted proposal.",
+    default=None,
+    help="The format is 'P[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{50}'",
+    validator=Validator([required_field_validator, protocol_hash_validator]),
+)
+
+# We define this step as a function since the corresponding step requires that we get the
+# proposal hashes off the chain.
+# tezos-client supports submitting up to 20 proposal hashes at a time, but it seems like this
+# isn't recommended for use with Ledger, so we leave it at one hash pro query for now.
+def get_proposal_period_hash(hashes):
+
+    # if the chain is in the proposal period, it's possible to submit a new proposal hash
+    extra_options = ["Specify new proposal hash"]
+
+    return Step(
+        id="chosen_hash",
+        prompt="Select a proposal hash.\n"
+        "You can choose one of the suggested hashes or provide your own:",
+        help="You can submit one proposal at a time.\n"
+        "'Specify new proposal hash' will ask a protocol hash from you. ",
+        default=None,
+        options=hashes + extra_options,
+        validator=Validator(
+            [
+                required_field_validator,
+                enum_range_validator(hashes + extra_options),
+            ]
+        ),
+    )
+
+
+ballot_outcome_query = Step(
+    id="ballot_outcome",
+    prompt="Choose the outcome for your ballot.",
+    help="'yay' is for supporting the proposal, 'nay' is for rejecting the proposal, "
+    "'pass' is used to not influence a vote but still contribute to reaching a quorum.",
+    options=ballot_outcomes,
```

Comment:
Let's add `default=None` here just for safety so that users cannot accidentally click 'Enter' and vote `yay`